Framework/CCDBSupport/src/CCDBHelpers.cxx

```diff
@@ -261,19 +308,7 @@ auto populateCacheWith(std::shared_ptr<CCDBFetcherHelper> const& helper,
         LOGP(detail, "******** Default entry used for {} ********", path);
       }
       helper->mapURL2UUID[path].lastCheckedTF = timingInfo.tfCounter;
-      if (etag.empty()) {
-        helper->mapURL2UUID[path].etag = headers["ETag"]; // update uuid
-        helper->mapURL2UUID[path].cachePopulatedAt = timestampToUse;
-        helper->mapURL2UUID[path].cacheMiss++;
-        helper->mapURL2UUID[path].minSize = std::min(v.size(), helper->mapURL2UUID[path].minSize);
-        helper->mapURL2UUID[path].maxSize = std::max(v.size(), helper->mapURL2UUID[path].maxSize);
-        api.appendFlatHeader(v, headers);
-        auto cacheId = allocator.adoptContainer(output, std::move(v), DataAllocator::CacheStrategy::Always, header::gSerializationMethodCCDB);
-        helper->mapURL2DPLCache[path] = cacheId;
-        O2_SIGNPOST_EVENT_EMIT(ccdb, sid, "populateCacheWith", "Caching %{public}s for %{public}s (DPL id %" PRIu64 ")", path.data(), headers["ETag"].data(), cacheId.value);
-        continue;
-      }
-      if (v.size()) { // but should be overridden by fresh object
```

Comment:
@shahor02 elsewhere we have a similar pattern, but using an `else if` rather than a separate else statement. Is that actually expected to be different here?